zerver/views/webhooks/crashlytics.py

```diff
@@ -0,0 +1,38 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+CRASHLYTICS_SUBJECT_TEMPLATE = '{display_id}: {title}'
+CRASHLYTICS_MESSAGE_TEMPLATE = '[Issue]({url}) impacts at least on {impacted_devices_count} device(s).'
+
+VERIFICATION_EVENT = 'verification'
+
+
+@api_key_only_webhook_view('Crashlytics')
+@has_request_variables
+def api_crashlytics_webhook(request, user_profile, client, payload=REQ(argument_type='body'),
+                            stream=REQ(default='crashlytics')):
+    try:
+        event = payload['event']
+        if event == VERIFICATION_EVENT:
+            return json_success()
```

Comment:
It's just a first event that checks if webhook works - I think we should handle it explicitly.
If webhook did not respond with 200 it means it does not work and crashlytics won't send any requests.